### DISCUSSION TITLE
Capacitor  3: "PushNotifications" plugin is not implemented on android

### DISCUSSION THREAD

**Discussion by jtejedera** [ionic-team/capacitor#4968] [Q&A]
Hello,

Trying  new  Capacitor 3 version on a fresh vue + capacitor project. On latest version of capacitor in other projects I worked with  PushNotifications with no problem, but  now in  v3 on the first implemention of the PushNotification for android, in Android Studio logs I  see: **Msg: Uncaught (in promise) Error: "PushNotifications" plugin is not implemented on android**
I installed the plugin as described in  the documentation and guide, sync the files, build the project...but anything seems to work.

[info] Found 5 Capacitor plugins for android:
       @capacitor/camera@1.0.4
       @capacitor/push-notifications@1.0.3
       @capacitor/splash-screen@1.1.0
       @capacitor/status-bar@1.0.2
       @codetrix-studio/capacitor-google-auth@3.0.2
√ update android in 93.94ms

In my  package.json:     "@capacitor/push-notifications": "^1.0.3",

I missed something?
Thank you.

**Comment by jtejedera** [comment]
Solved, my fault, removed 
```
this.init(savedInstanceState, new ArrayList<Class<? extends Plugin>>() {{
// Additional plugins you've installed go here
add(Plugin1.class);
add(Plugin2.class);
}});
```

from MainActivity.java

**Reply by johnborges** [reply]
@jtejedera Was `MainActivity.java` populated automatically for you, or did you need to add your plugins their manually?
